ddl/serial_test.go

```diff
@@ -1409,3 +1409,24 @@ func (s *testSerialSuite) TestCreateClusteredIndex(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(tbl.Meta().IsCommonHandle, IsFalse)
 }
+
+func (s *testSerialSuite) TestCreateTableNoBlock(c *C) {
+	tk := testkit.NewTestKitWithInit(c, s.store)
+	c.Assert(failpoint.Enable("github.com/pingcap/tidb/ddl/mockCreateTableCheckFail", `return(true)`), IsNil)
+	defer func() {
+		c.Assert(failpoint.Disable("github.com/pingcap/tidb/ddl/mockCreateTableCheckFail"), IsNil)
+	}()
+	c.Assert(failpoint.Enable("github.com/pingcap/tidb/ddl/checkOwnerCheckAllVersionsWaitTime", `return(true)`), IsNil)
```

Comment:
Could we use one failpoint like `mockCreateTableFailCheckSchemaVersion` instead of `mockCreateTableCheckFail` and `checkOwnerCheckAllVersionsWaitTime `?

Reply:
Done